pkg/kubelet/winstats/winstats_test.go

```diff
@@ -123,6 +130,23 @@ func TestWinVersionInfo(t *testing.T) {
 		KernelVersion: "v42"})
 }
 
+func convertCPUValue(cpuValue uint64) uint64 {
+	cpuCores := 4
+	f := fakeWinNodeStatsClient{}
+	nodeMetrics, err := f.getNodeMetrics()
+	if err != nil {
+		return 0
+	}
+	newValue := nodeMetrics.cpuUsageCoreNanoSeconds + uint64((float64(cpuValue)/100.0)*float64(cpuCores)*1e9)
```

Comment:
Rather than copy the calculation here, the test should call the same function used by the application code so they can't diverge.

Reply:
Done.